scripts/build.py

```diff
@@ -430,5 +570,18 @@ def generate_build_directory():
         TEMPLATES_STAGING_DIR, TEMPLATES_OUT_DIR, hashes)
 
 
+def build():
+    parser = optparse.OptionParser()
+    parser.add_option(
+        '--prod_env', action='store_true', default=False, dest='prod_mode')
+    options = parser.parse_args()[0]
+
+    print options.prod_mode
```

Comment:
Print something that's more human-readable as an English sentence. See the backend test scripts, for example, or the startup scripts.

E.g. "Building Oppia in production mode (minifying files)..." and then "Build completed."

Or "Re-building Oppia in dev mode" (maybe give the user some idea of what that actually means).

Basically, it's worth thinking through the dev experience and making sure that devs have enough understanding of what's going on without needing to introspect this script.

Reply:
Done.